solvers/blitz.py

```diff
@@ -32,9 +32,9 @@ def set_objective(self, X, y, lmbd, fit_intercept):
         blitzl1.set_use_intercept(False)
         self.problem = blitzl1.LassoProblem(self.X, self.y)
 
-    def run(self, tolerance):
-        blitzl1.set_tolerance(tolerance)
-        self.coef_ = self.problem.solve(self.lmbd).x
+    def run(self, n_iter):
+        blitzl1.set_tolerance(0)
```

Comment:
Shouldn't we set this in `set_objective`?

Reply:
Indeed, done !